vital/utils/saving.py

```diff
@@ -114,32 +114,33 @@ def load_from_checkpoint(
         train_mode: Whether the model should be in 'train' mode (`True`) or 'eval' mode (`False`).
         device: Device on which to move the Lightning module after it's been loaded. Defaults to using 'cuda' if it is
             available, and 'cpu' otherwise.
-        expected_checkpoint_type: Type of model expected to be loaded from the checkpoint. Used to perform a runtime
-            check, and raise an error if the expected model type does not match the loaded model.
+        expected_checkpoint_type: Type of system expected to be loaded from the checkpoint. Used to perform a runtime
+            check, and raise an error if the expected system type does not match the loaded system.
 
     Returns:
-        Lightning module loaded from the checkpoint, casted to its original type.
+        Lightning module (specifically a subclass of VitalSystem) loaded from the checkpoint, casted to its original
```

Comment:
While we're at it, let's correct a verb tense mistake I left behind previously (the past tense of _to cast_ is simply _cast_):
```suggestion
        Lightning module (specifically a subclass of VitalSystem) loaded from the checkpoint, cast to its original
```